fs/unpacker_test.go

```diff
@@ -210,3 +213,142 @@ func getFakeMounts() []mount.Mount {
 		},
 	}
 }
+
+// Test that AsyncTeeReader copies data to both the returned reader and the writer.
+func TestAsyncTeeReaderBasic(t *testing.T) {
```

Comment:
Should we combine this with the test below? They look to do the same thing.

Reply:
updated